src/test/java/nl/ordina/jobcrawler/service/SkillServiceTest.java

```diff
@@ -0,0 +1,298 @@
+package nl.ordina.jobcrawler.service;
+
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.repo.SkillRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+class SkillServiceTest {
```

Comment:
SkillService.updateSkills() is not fully covered.

Reply:
the last 3 methods (updateSkills_deletingSkill, updateSkills_addExistingSkill, updateSkills_addNewSkill) are testing different cases of SkillService.updateSkills()